modules/shared.py

```diff
@@ -123,6 +123,7 @@ def str2bool(v):
 parser.add_argument('--n_batch', type=int, default=512, help='Maximum number of prompt tokens to batch together when calling llama_eval.')
 parser.add_argument('--no-mmap', action='store_true', help='Prevent mmap from being used.')
 parser.add_argument('--mlock', action='store_true', help='Force the system to keep the model in RAM.')
+parser.add_argument('--n_gpu_layers', type=int, default=0, help='Number of layers to offload to the GPU.')
```

Comment:
```suggestion
parser.add_argument('--n-gpu-layers', type=int, default=0, help='Number of layers to offload to the GPU.')
```
To match llama.cpp argument - 
![image](https://github.com/oobabooga/text-generation-webui/assets/48323879/2fb2f912-25e9-46fc-9bfa-3dd4ee11626d)